diff --git a/node_modules/hexo-theme-next/layout/_partials/footer.njk b/node_modules/hexo-theme-next/layout/_partials/footer.njk
index 0e27c12..cd9c91d 100644
--- a/node_modules/hexo-theme-next/layout/_partials/footer.njk
+++ b/node_modules/hexo-theme-next/layout/_partials/footer.njk
@@ -77,7 +77,11 @@
 {%- if theme.footer.powered %}
   <div class="powered-by">
     {%- set next_site = 'https://theme-next.js.org' if theme.scheme === 'Gemini' else 'https://theme-next.js.org/' + theme.scheme | lower + '/' %}
-    {{- __('footer.powered', next_url('https://hexo.io', 'Hexo') + ' & ' + next_url(next_site, 'NexT.' + theme.scheme)) }}
+    {{- __('footer.powered', next_url(next_site, 'NexT')) }}
+    &
+    <a href='https://www.upyun.com/?utm_source=lianmeng&utm_medium=referral' rel="noopener external nofollow noreferrer" target="_blank" class="upyan" >
+      <img src="https://files.zuiyu1818.cn/upyun.png" alt="又拍云">
+    </a>提供加速
   </div>
 {%- endif %}
 
